manifests/0000_50_olm_01-olm-operator.serviceaccount.yaml

```diff
@@ -28,6 +28,7 @@ rules:
       - securitycontextconstraints
     resourceNames:
       - restricted-v2
+      - anyuid
```

Comment:
This is needed to satisfy a feature in which we allow catalogSources in legacy mode, which cannot run under the constraints introduced by the `restricted-v2` SCC, you can learn more by inspecting [this test](https://github.com/openshift/operator-framework-olm/blob/2ff0c6731160ed93ae9d356e1d1c50e2bafaa06a/staging/operator-lifecycle-manager/test/e2e/catalog_e2e_test.go#L1540-L1612).